2/contrib/jenkins/install-plugins.sh

```diff
@@ -310,7 +317,7 @@ function resolveDependencies() {
             # download the dependence; passing "true" is needed for "download" to replace the existing dependency
             if versionLT "${versionInstalled}" "${minVersion}"; then
                 echo "Upgrading bundled dependency $d ($minVersion > $versionInstalled)"
-                download "$plugin" "$minVersion" "true"
+                download "$plugin" "$minVersion"
```

Comment:
Was the bug here? I don't see a signature change to the download function, however, it was used only with 2 arguments previously, the 2nd one being a boolean. no regression ?

Reply:
Since this was passing `true`, it was overriding the locked versions of plugins and potentially installed a different version, which may or may not be correct, or what we wanted.